TwoFactorAuth/view/adminhtml/web/js/registry.js

```diff
@@ -3,12 +3,18 @@
  * See COPYING.txt for license details.
  */
 
-'use strict';
+define(
+    [],
+    function () {
+    'use strict';
 
-define([
-    'ko'
-], function (ko) {
     return {
-        trustThisDevice: ko.observable(false)
+        /**
+         * Trusted devices are not allowed.
+         * @returns {Boolean}
+         */
+        trustThisDevice: function () {
```

Comment:
continuing the commentary on the nature of this deprecation, this is not a BC way to replace a ko object and will break anybody trying to subscribe to the value changes via `subscribe`.